client/extensions/woocommerce/state/sites/payment-methods/actions.js

```diff
@@ -107,3 +100,47 @@ export const savePaymentMethod = ( siteId, method, successAction = null, failure
 			}
 		} );
 };
+
+const savePaymentMethodEnabledSuccess = ( siteId, data ) => {
+	const paymentMethod = addPaymentMethodDetails( data );
+	return {
+		type: WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE_SUCCESS,
+		siteId,
+		data: paymentMethod,
+	};
+};
+
+export const savePaymentMethodEnabled = (
+	siteId,
+	method,
+	enabled,
+	successAction = null,
+	failureAction = null
+) => ( dispatch, getState ) => {
+	const state = getState();
+	if ( ! siteId ) {
+		siteId = getSelectedSiteId( state );
```

Comment:
I could change it but if save is being called we definitely have the site id.  There are no payment methods to save if you don't have the site id.